pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -112,6 +112,7 @@ func defaultConfigMapData() map[string]string {
 		"scale-to-zero-grace-period":              gracePeriod.String(),
 		"tick-interval":                           "2s",
 		"min-scale":                               "0",
+		"target-burst-capacity":                   "200",
```

Comment:
Shouldn't this be 211 then now, too ?

Reply:
I borrowed this from #12241 which didn't change the TBC value for the tests. 

Before I saw that, I tried using TBC=211 and it caused a bunch of the tests to fail. I didn't dig too deep into it, so it could be that the tests are tuned to the 200 value (though that's just speculation, I really didn't look into for more than a few minutes).

If we'd like to get the tests passing with the updated TBC value, I can work on that (one thing I did find was increasing the minActivators by one (2 -> 3) fixed two-thirds of the errors, though not sure why exactly).